README.md

```diff
@@ -35,7 +35,8 @@ dependency on and install with the following command:
 go get github.com/segmentio/parquet-go
 ```
 
-Go 1.17 or later is required to use the package.
+Go 1.17 is required to use the package, but some APIs and optimizations are only
```

Comment:
```suggestion
Go 1.17 is the lowest supported version. Some APIs and optimizations are only
```

Reply:
Previously you might think you _had_ to use 1.17 - saying it's "required" doesn't make sense in context if you are using 1.18

Reply:
I can rephrase 👍